ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -424,8 +424,16 @@ Redundancy:
 snmp_server:
   contact: < contact_name >
   location: < location >
-  ipv4_access_list: < ipv4-access-list >
-  ipv6_access_list: < ipv6-access-list >
+  ipv4_ACLs: 
```

Comment:
Maybe use lowercase everywhere, as that's how we do everything

Reply:
Done. Thanks for the catch up!